Documentation/data-collection.md

```diff
@@ -205,6 +205,10 @@ data:
     # (monitoring-team) monitoring:haproxy_server_http_responses_total:sum tracks the number of times users access
     # monitoring routes.
     - '{__name__="monitoring:haproxy_server_http_responses_total:sum"}'
+    # (rhmi, @openshift/openshift-team-cluster-manager) rhmi_status reports the status of an RHMI installation.
```

Comment:
Can you be in multiple stages?  If not, why not only report a single metric?

Reply:
> Can you be in multiple stages?

No.

> If not, why not only report a single series?

I may be misunderstanding how labels impact the number of series, but do you mean only ever expose 1 metric result from the scrape endpoint with whatever the current stage is in the stage label (with a value of 1)? 

e.g.
* from 11:30 to 11:35 `rhmi_status{stage="bootstrap"} 1`
* from 11:35 to 11:39 `rhmi_status{stage="products"} 1`
* after 11:39 `rhmi_status{stage="complete"} 1`

Or use the value to represent the stage (no label)?

e.g.
* from 11:30 to 11:35 `rhmi_status{} 1`
* from 11:35 to 11:39 `rhmi_status{} 2`
* after 11:39 `rhmi_status{} 3`

Reply:
The first one - you don't need 8 series with 1 set to 1, and the other 7 set to 0, if the question you are primarily answering is "how many clusters are in this state", "is this cluster in this state".  The zero values don't convey any additional info over their absence as far as I can tell

```

    from 11:30 to 11:35 rhmi_status{stage="bootstrap"} 1
    from 11:35 to 11:39 rhmi_status{stage="products"} 1
    after 11:39 rhmi_status{stage="complete"} 1
```

Reply:
> you don't need 8 series with 1 set to 1

Good point. 
My thinking was trying to mimic what kube-state-metrics does for some metrics like `kube_pod_status_phase` e.g.

```
kube_pod_status_phase{...,phase="Failed",...} | 0
kube_pod_status_phase{...,phase="Pending",...} | 0
kube_pod_status_phase{...,phase="Running",...} | 1
kube_pod_status_phase{...,phase="Succeeded",...} | 0
kube_pod_status_phase{...,phase="Unknown",...} | 0
```

I'll update the metric upstream in https://github.com/integr8ly/integreatly-operator/pull/857 to omit the 0's.

Reply:
True, if 0 values you don't care about, you could do a recording rule that checks anything` larger than 0` or `== 1`, and just send the recording rule via telemetry, that will indeed save you few series per cluster.

Reply:
Confirming I've updated the metric being exposed so it doesn't return 0 values (https://github.com/integr8ly/integreatly-operator/pull/857)